addOns/ascanrules/CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Update dependency.
 - CVE-2012-1823 Remote Execution and Source Code Disclosure, and Heart Bleed scan rules now include Alert Tags for the applicable CVEs.
 
+### Added
+- ParameterEntity attack added in OOB-XXE Scan Rule.
```

Comment:
This needs to be moved to the Unreleased section.

Reply:
Done